readthedocs/search/faceted_search.py

```diff
@@ -79,7 +84,8 @@ def _get_text_query(self, *, query, fields, operator):
                 query=query,
                 fields=fields,
                 operator=operator,
-                fuzziness="AUTO",
+                fuzziness="AUTO:4,6",
+                prefix_length=1,
```

Comment:
This one made a lot of difference

Reply:
Yea, I think that will make it feel much less random.